Providers/Pocket/PocketFeed.swift

```diff
@@ -100,7 +109,9 @@ class Pocket: FeatureFlaggable, URLCaching {
             params.append(URLQueryItem(name: "consumer_key", value: pocketKey))
         }
 
-        guard let feedURL = URL(string: pocketGlobalFeed)?.withQueryParams(params) else { return nil }
+        guard let feedURL = URL(string: pocketGlobalFeed)?.withQueryParams(params) else {
+            return nil
+        }
```

Comment:
```suggestion
        guard let feedURL = URL(string: pocketGlobalFeed)?.withQueryParams(params) else { return nil }
```
Minor nit. Unless the guard or it's else goes on too long, or it has multiple parameters we should try and keep them to the same line for cleanliness. Doesn't fully matter, but just a matter of style. :)